src/core/global-api.js

```diff
@@ -1,10 +1,10 @@
-import prism from 'prismjs';
-import marked from 'marked';
 import * as util from './util';
 import * as dom from './util/dom';
 import { Compiler } from './render/compiler';
 import { slugify } from './render/slugify';
 import { get } from './fetch/ajax';
+import marked from 'marked';
```

Comment:
is this also autofixed ?

Reply:
The reordering changes in commit [7fe044bcdeba6a1f4a5026f53890dad7087a784f](https://github.com/docsifyjs/docsify/pull/1311/commits/7fe044bcdeba6a1f4a5026f53890dad7087a784f) is all auto-fixed.

Reply:
@anikethsaha I found a weird thing, that my local linting does not tell this as warnings or errors, but GitHub Actions workflow tells.
![image](https://user-images.githubusercontent.com/3257783/91644380-0b977980-ea6e-11ea-9c9e-ecaf11fe1cba.png)

Reply:
After I moved `marked` line to top, and run `npm run lint`, it shows as following:
![image](https://user-images.githubusercontent.com/3257783/91644430-78127880-ea6e-11ea-8bca-dd067ec51514.png)

Reply:
leave the warnings

Reply:
Should I revert all such order changes due to `--fix` option?

Reply:
No need to revert as long as they are warning.

Reply:
@anikethsaha It is weird that the default order for `import/order` plugin is different for my local and the GitHub Actions environment. In the new commit, I fixed the order in `eslintrc.js` (according to [plugin doc](https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md)).

Reply:
Yeah there are some inconsistency in that place. Thanks for fixing it.